src/doom/info.c

```diff
@@ -1229,7 +1229,10 @@ mobjinfo_t mobjinfo[NUMMOBJTYPES] = {
 	0,		// damage
 	sfx_None,		// activesound
 	MF_SOLID|MF_SHOOTABLE|MF_DROPOFF|MF_PICKUP|MF_NOTDMATCH|MF_FLIPPABLE,		// flags
-	S_NULL		// raisestate
```

Comment:
Her, the same concerns as in the previous PR apply: How should I seriously review these 1000+ lines of code changes and what about the 100 additional extra states that are not initialized in this array?